gateway/1.4/modules/ROOT/pages/flex-gateway-getting-started-kubernetes.adoc

```diff
@@ -0,0 +1,650 @@
+= Getting Started with Flex Gateway on a Kubernetes Cluster
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:version-helm: 3.0.0
+
+Get started with Anypoint Flex Gateway on a Kubernetes cluster. Set up a Flex Gateway deployment that connects to Anypoint Platform, and use the deployment to route traffic to your API. This guide covers the following high-level tasks:
+
+* Downloading, registering, and deploying Flex Gateway in Connected Mode. 
```

Comment:
Sentence fragments with periods.

Reply:
Do we not allow fragments in bullet lists? Totally though it was kosher.

Reply:
Fragments are fine, but if they're all fragments, no periods.

Reply:
https://confluence.internal.salesforce.com/pages/viewpage.action?spaceKey=MTDT&title=MuleSoft+CX+Writing+Style+Reference#MuleSoftCXWritingStyleReference-bulletlists